deliver/lib/deliver/runner.rb

```diff
@@ -281,20 +281,20 @@ def transporter_for_selected_team(upload: false)
 
       unless api_token.nil?
         api_token.refresh! if api_token.expired?
-        return FastlaneCore::ItunesTransporter.new(nil, nil, false, nil, api_token.text, upload: upload, api_key: api_key)
+        return FastlaneCore::ItunesTransporter.new(nil, nil, false, nil, api_token.text, altool_compatible_command: altool_compatible_command, api_key: api_key)
```

Comment:
We can always pass `altool_compatible_command: true` because of the above comment.